packages/client/components/AtmosphereProvider/PortalProvider.tsx

```diff
@@ -3,7 +3,9 @@ import {createPortal} from 'react-dom'
 import useForceUpdate from '../../hooks/useForceUpdate'
 
 export type SetPortal = (id: string, portal: ReactNode) => void
-export const PortalContext = React.createContext((() => {}) as SetPortal)
+export const PortalContext = React.createContext((() => {
+  return
```

Comment:
Currently, [`no-empty-function`](https://typescript-eslint.io/rules/no-empty-function/) is enforced. So, to retain the existing behavior and make this rule happy, I `return`.

Reply:
+1 I prefer to put a comment `// noop` there to emphasize it's intentional